### PR TITLE
{bio}[foss/2018b] CD-HIT v4.8.1

### PR DESCRIPTION
**Pull request by jdemeul**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.8.1-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# make sure compilation flags are passed down (e.g. to enable OpenMP)
+buildopts = ' CC="$CXX" CCFLAGS="$CPPFLAGS $CXXFLAGS"'
+
+# put here the list of generated executables when compiling
+list_of_executables = ["cd-hit", "cd-hit-est", "cd-hit-2d", "cd-hit-est-2d", "cd-hit-div", "cd-hit-454"]
+
+# this is the real EasyBuild line to copy all the executables and perl scripts to "bin"
+files_to_copy = [(list_of_executables, "bin"), (["*.pl"], 'bin'), "README", "doc", "license.txt"]
+
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.8.1-foss-2018b.eb]: is 
```
postinstallcmds = ["sed -i 's@#!/usr/bin/perl@/usr/bin/env perl@' %(installdir)s/bin/*.pl"]
```
no longer necessary?

Comment (boegel) [reply easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.8.1-foss-2018b.eb]: It's still required, I'll add it back and get this one merged...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3118.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/93db7cdbabe93b7416cc8b5734c3df7b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2039.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/72a43a74ff76ce86f1876b28397c2dcd for a full test report.

**Comment by boegel** [comment]
Going in, thanks @jdemeul!

**Pull request merged**